jupyter-extension/src/plugins/mount/mount.tsx

```diff
@@ -614,20 +459,16 @@ export class MountPlugin implements IMountPlugin {
     await this._poller.refresh();
 
     if (this._poller.status.code === 500) {
-      this.setShowFullPageError(true);
+      console.error('poller status 500');
+      // this.setShowFullPageError(true);
```

Comment:
Maybe `import {showErrorMessage} from '@jupyterlab/apputils';` can be used here like in `customFileBrowser.ts#118`? It creates a popup on screen for the user to read then dismiss.

Reply:
Ah nice! I've always been annoyed by only having "FullPageError" at our disposal for these situations. I'll probably make the substitution you suggest here and then make a separate ticket to address these error states